### PR TITLE
Project and Run Charts

### PR DESCRIPTION
**Pull request by jorgemiguelreis**

### CODE REVIEW

File: src/views/partials/includes.html.tpl [hunk 1]
```diff
@@ -1,2 +1,3 @@
 <link rel="shortcut icon" href="" />
 <script src="/js/bundle.js"></script>
+<script src= "https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.7.3/Chart.min.js"></script>
```
Comment (marcosoliveira91) [inline src/views/partials/includes.html.tpl]: don't need this, npm is including the lib already

File: src/utils/chartBuilder.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+module.exports = {
```
Comment (marcosoliveira91) [inline src/utils/chartBuilder.js]: is this for debug only? If so, it's better to not merge it

File: src/views/partials/includes.html.tpl (hunk 1 above)
Comment (jorgemiguelreis) [reply src/views/partials/includes.html.tpl]: 👍

File: src/utils/chartBuilder.js (hunk 2 above)
Comment (jorgemiguelreis) [reply src/utils/chartBuilder.js]: No longer used 👍

**Pull request merged**